jetty-server/src/main/java/org/eclipse/jetty/server/ResourceService.java

```diff
@@ -669,18 +672,14 @@ protected boolean sendData(HttpServletRequest request,
             if (include)
             {
                 // write without headers
-                content.getResource().writeTo(out, 0, content_length);
+                writeContent(content, out, 0, content_length);
             }
             // else if we can't do a bypass write because of wrapping
             else if (written || !(out instanceof HttpOutput))
```

Comment:
Nothing really to do with the changes but this line could be cleaned up, the value of `!(out instanceof HttpOutput)` is always false.